core/src/main/java/nu/yona/server/subscriptions/entities/UserAnonymized.java

```diff
@@ -37,12 +41,14 @@ public static UserAnonymizedRepository getRepository()
 
 	private LocalDate lastMonitoredActivityDate;
 
-	@OneToOne
+	@OneToOne(fetch = FetchType.LAZY)
 	private MessageDestination anonymousDestination;
 
 	@OneToMany(mappedBy = "userAnonymized", cascade = CascadeType.ALL, orphanRemoval = true)
 	@Where(clause = "end_time is null") // The history items have the user anonymized ID set, so they would appear in this
 										// collection if not explicitly excluded
+	@Fetch(FetchMode.SELECT)
```

Comment:
This is the default, so does not have to be here. See [Hibernate documentation](http://docs.jboss.org/hibernate/core/4.3/manual/en-US/html/ch20.html#performance-fetching-lazy).